### ISSUE TITLE
Are two json libraries really needed?

### ISSUE THREAD

**Issue by acwwat** [google/identity-toolkit-java-client#33]
The project uses both gson and json, although gson can probably do everything that json can. Is there any reason why both are needed, and if not, can json be removed so to reduce dependencies?

**Comment by liujin-google** [comment]
Good point. I think the dependency on org.json is unnecessary and can be dropped. Pull requests are welcome :)

**Comment by acwwat** [comment]
See pull request #39. Note that I list some concerns for review.
